src/auth/user-record.ts

```diff
@@ -56,6 +56,13 @@ export interface TotpInfoResponse {
   [key: string]: unknown;
 }
 
+export interface PasskeyInfoResponse {
+  name?: string;
+  credentialId?: string;
+  displayName?: string;
+  [key: string]: unknown;
```

Comment:
What's this field used for?

Reply:
I followed convention looking at `MultiFactorInfoResponse` and `GetAccountInfoUserResponse`. Can be removed if we think it is redundant.

Reply:
Yeah, it's useful for parsing unknown fields from backend which might not be the case here. Let's add it back when needed.